### PR TITLE
Refactoring condition

### PR DESCRIPTION
**Pull request by GitHubHubus**
1) Moved conditions in a separate class to avoid swelling `JsonQueryable` trait and to get rid of the prefix `cond` for condition methods.
2) Made more readable `processConditions` function. Without logic change.

### CODE REVIEW

**Comment by peter279k** [comment]
It seems that the Travis CI build is broken due to the PHP package dependencies.

It should be fixed.

File: src/JsonQueriable.php [hunk 1]
```diff
@@ -248,7 +249,7 @@ protected function getData()
         if (empty($this->_node) || $this->_node == '.') {
             return $this->_map;
         }
-
+
```
Comment (peter279k) [inline src/JsonQueriable.php]: It looks like that is the additional white space.
It should be removed.

File: src/Condition.php [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+    /**
+     * Strict greater than
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function greater($value, $comparable)
```
Comment (peter279k) [inline src/Condition.php]: Why not named the method ```greaterThan````?

File: src/Condition.php [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+    /**
+     * Strict less than
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     * 
+     * @return bool
+     */
+    public static function condLess($value, $comparable)
```
Comment (peter279k) [inline src/Condition.php]: Why not named the ```lessThan```?

File: src/Condition.php [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+    /**
+     * Greater or equal
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function greaterEqual($value, $comparable)
```
Comment (peter279k) [inline src/Condition.php]: How about ```greaterThanOrEqual```?

File: src/Condition.php [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+    /**
+     * Less or equal
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function lessEqual($value, $comparable)
```
Comment (peter279k) [inline src/Condition.php]: How about ```lessThanOrEqual```?

File: src/Condition.php (hunk 2 above)
Comment (GitHubHubus) [reply src/Condition.php]: I think, you are right about all cases. Thanks.

**Pull request merged**